Samples.Objc/Samples.ObjC/ViewControllers/AddItemViewController/AddItemViewController.m

```diff
@@ -0,0 +1,222 @@
+//
+//  AddItemViewController.m
+//  PayPalNativeCheckoutObjC
+//
+//  Created by Nguyen, The Nhat Minh on 3/9/21.
+//
+
+#import "AddItemViewController.h"
+#import "TextFieldCell.h"
+#import "QuantityView.h"
+
+@interface AddItemViewController ()
+@property (nonatomic) NSString *itemName;
+@property (nonatomic) NSString *itemPrice;
+@property (nonatomic) NSString *titleText;
+@property (nonatomic) NSString *buttonTitle;
+@property (nonatomic) UIView *containerView;
+@property (nonatomic) UILabel *titleLabel;
+@property (nonatomic) UITableView *tableView;
+@property (nonatomic) QuantityView *quantityView;
+@property (nonatomic) UIButton *saveButton;
+@property (nonatomic) NSLayoutConstraint *containerViewBottom;
```

Comment:
Are we able to add a tax price here as well? This will match up with Android and Swift samples. Will need to update the values passed into the order request as well to support this.

Reply:
I agree!!!! I just added tax price :smile: